core/App/screens/ListContacts.tsx

```diff
@@ -15,7 +15,7 @@ const ListContacts: React.FC<ListContactsProps> = ({ navigation }) => {
   const { records } = useConnections()
   // Filter out mediator agents
   const connections = records.filter((r) => {
-    const type = (r.getTag('connectionType') || []) as [string]
+    const type = (r.getTag('connectionTypes') || []) as [string]
```

Comment:
I believe we no longer need to call the `getTag` method. There should be an getter method directly on the record now.

https://github.com/hyperledger/aries-framework-javascript/pull/1177

https://github.com/hyperledger/aries-framework-javascript/blob/4a572fedd7508aeba3e91f4d1d98b9a177941648/packages/core/src/modules/connections/repository/ConnectionRecord.ts#L68

The migration scripts in AFJ 0.3.0 should have moved those tags over

Reply:
Thanks for the tip. Just checked the code and used the accessor.